### ISSUE TITLE
User story - Tech Aliases

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#40]
### User Story

As a **tech user**, I can **add a tech by its other name without affecting its semantic meaning** so that I can **have flexibility when adding tech**.

### Acceptance Criteria

- The user should be able to add tech by another name, e.g. CSS is the same as CSS3 or Cascading Stylesheets

### Please ensure you've completed these tasks before marking the issue as done.

- [ ] Reviewed the user story for clarity and completeness.
- [ ] Identified and linked related issues or epics.
- [ ] Created or updated the necessary documentation.
- [ ] Written unit and/or integration tests for functionality.
- [ ] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [ ] Updated the issue with any relevant notes or feedback (to date).
- [ ] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [ ] Add a new one to many table for tech aliases for each tech
